src/kiutils/symbol.py

```diff
@@ -285,10 +289,16 @@ def from_sexpr(cls, exp: list):
             if item[0] == 'power': object.isPower = True
 
             if item[0] == 'symbol':
-                unit_symbol = Symbol().from_sexpr(item)
-                # Store unit numbers
-                unit_symbol.unit_numbers = (unit_symbol.id.split('_')[-2], unit_symbol.id.split('_')[-1])
-                object.units.append(unit_symbol)
+                # Get symbol unit 
```

Comment:
Instead of these small comments, better combine them to one describing what this code block does. Something like this:

> Create a new subsymbol and parse its unit and style identifiers from the combined ID field